src/include/daos_errno.h

```diff
@@ -262,7 +262,10 @@ extern "C" {
 	       TX status is uncertaion)					\
 	/** Communicatin issue with agent. */				\
 	ACTION(DER_AGENT_COMM,		(DER_ERR_DAOS_BASE + 34),	\
-		Agent communication error)
+	       Agent communication error)				\
+	/** ID mismatch */						\
+	ACTION(DER_ID_MISMATCH,		(DER_ERR_DAOS_BASE + 35),	\
```

Comment:
technically, this should be a minor API version bump

Reply:
OK. On release/1.2, the API version is 1.2.0; on master, the API version is currently at 1.5.0. So we don't need to make any further changes for DAOS v2.0?